package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "koa-middlewares",
-  "version": "2.1.0",
+  "version": "2.2.0",
```

Comment:
you should not change the version in PR, I'll release a new version after merge this PR. thanks